contrib/guix/libexec/prelude.bash

```diff
@@ -50,7 +50,7 @@ fi
 # across time.
 time-machine() {
     # shellcheck disable=SC2086
-    guix time-machine --url=https://git.savannah.gnu.org/git/guix.git \
+    guix time-machine --url=https://codeberg.org/guix/guix-mirror.git \
```

Comment:
Why switch to a [mirror](https://lists.gnu.org/archive/html/info-guix/2025-02/msg00000.html) set up three months ago rather then wait a bit and switch to https://codeberg.org/guix/guix.git, as described in https://issues.guix.gnu.org/76503?

Reply:
This is a draft, and the repository you've linked to doesn't exist yet.

Reply:
A  new https://git.guix.gnu.org/guix.git seems up.